justfile

```diff
@@ -49,3 +49,9 @@ schema-modules:
     (cd $path; cargo run --example schema --features schema); 
   done
   set +e
+
+schema: schema-modules
```

Comment:
We should have one for framework as well

Reply:
there's a cargo-workspace command that lets you run commands on every member of a workspace.

Reply:
https://crates.io/crates/cargo-workspaces

Reply:
See the justfile in modules!

Reply:
```
# TODO this doesn't work for contracts like the croncat one which is the croncat prefix
publish-schemas:
  SCHEMA_OUT_DIR=$(cd ../../schemas && echo "$PWD") \
    cargo ws exec --no-bail bash -lc 'cargo schema && { jq -r .contract_version schema/module-schema.json > _version.txt; outdir="$SCHEMA_OUT_DIR/abstract/${PWD##*/}/$(cat _version.txt)"; mkdir -p "$outdir"; rm -rf "schema/raw"; cp -a "schema/." "$outdir"; rm _version.txt; }'

```

Reply:
> ```
> # TODO this doesn't work for contracts like the croncat one which is the croncat prefix
> publish-schemas:
>   SCHEMA_OUT_DIR=$(cd ../../schemas && echo "$PWD") \
>     cargo ws exec --no-bail bash -lc 'cargo schema && { jq -r .contract_version schema/module-schema.json > _version.txt; outdir="$SCHEMA_OUT_DIR/abstract/${PWD##*/}/$(cat _version.txt)"; mkdir -p "$outdir"; rm -rf "schema/raw"; cp -a "schema/." "$outdir"; rm _version.txt; }'
> ```

There's similar script to this one in framework justfile, we can execute that, if we don't plan to remove justfile's

Reply:
Yeah, but I don't even know what that command does. 
I think we only need to publish schemas for the modules but not sure. 

@adairrr

Reply:
It runs cargo schema for each of the modules, then copies the contents of the schemas for those modules to the proper directory (name + version) in the schemas folder (which in writing the command was in the `../../schemas`. It's a bit hacky and we should move it to a github action.

Reply:
I say we move it to a `.sh` script and call that from the github action. Easier to maintain and we can run it locally as well. 

@Buckram123 Could you do that?

Reply:
Do we want copy schemas to `../../schemas` in the same script?

Reply:
I don't know, should all the schemas live in to one dir? What's the desired setup here? 
@adairrr